### DISCUSSION TITLE
Dziś jest...(riposta)

### DISCUSSION THREAD

**Discussion by zaba-monika** [MajkiIT/polish-ads-filter#17640] [General] [answered]
Pajace założyli sobie głupią dyskusję  i trolują mnie: https://github.com/MajkiIT/polish-ads-filter/discussions/17449#discussioncomment-114218 ,
dobrze wiedząc, że nie mogę się obronić / zripostować im na te ich wypociny:

![](https://user-images.githubusercontent.com/72980128/99354731-48be1b80-28a7-11eb-8955-10cd6c45790d.png)

- więc takim swoim durnym i chamskim zachowaniem zmuszają mnie do założenia swojej dyskusji i zripostowania im:

Riposta:

> https://github.com/MajkiIT/polish-ads-filter/discussions/17449#discussioncomment-114218 : hawkeye116477 : 
A dzisiaj jest coś co zainteresuje @zaba-monika - **Dzień Odpoczynku dla Zszarganych Nerwów.**
>> xxcriticxx : na to jest clonazepam

-  trollu nie obsmarowywuj kogoś, gdy nie dajesz mu możliwości obronienia się i zripostowania na twoje wypociny, bo to jak kopanie leżącego, robiłeś tak już nie raz, nie posiadasz i nigdy nie będziesz posiadał jakiegokolwiek honoru czy zasad, bo trolle ich nie mają
- nie obrażajcie kobiety
- coś sobie wymyśliliście na potrzeby trolowania, nie wskazaliście o niby jakie zszargane nerwy wam chodzi, trolujecie ogólnikowo bez wskazania / zacytowania konkretnego komentarza, zdania, słów..widać, że wasze zarzuty są wyssane z palca, jakaś chyba bajka, i chodzi wam po prostu o to, żeby sobie potrolować, nawet nie wiem do czego mam się odnieść, a do bajek i wymysłów nie ma sensu się odnosić...
- 
> hawkeye116477 : **Dzień Odpoczynku dla Zszarganych Nerwów.**
> xxcriticxx : na to jest clonazepam

jak to zwykle u was, hipokryzja jak zawsze, bo wytykacie komuś zszargane nerwy (które gdzie niby są?) po tym gdy sami wpierw te nerwy komuś próbujecie zszargać + skoro komuś bez powodu nerwy próbujecie zszargać i atakujecie to świadczy też o tym, że sami macie zszargane...więc sami sobie łyknijcie clonazepam, a ten dzień wymyślono właśnie przez takich trolli jak ty i twój przydupas xxcritcicxx, którzy próbują szargać innym nerwy, więc rzeczywiście czasem mi się przyda odpoczynek od takich jak wy

- dopiero co kilka dni temu razem ze swoim przydupasem xxcriticxx już raz mnie prowokowaliście w [TYM WĄTKU](https://github.com/MajkiIT/polish-ads-filter/issues/17587#issuecomment-725452043) żeby tylko mieć pretekst do obwinienia mnie o cokolowiek i zbanowania mnie, ale na spokojnie zripostowałam was, a admin przejrzał na oczy, że mnie zaczepiacie i prowokujecie + że ja tu dużo pomagam, więc wasze marzenie o bezpodstawnym zbanowaniu mnie się nie ziściło, bo przynajmniej admin miał trochę rozsądku, i tym razem też tak będzie, bo znowu spokojnie ripstuję wasze prowokacje + dużo pomagam, więc admin tym razem też nie wysłucha waszych wypocin w moją stronę

- takich prób zszargania mi nerwów oraz prowokacji / trolowania / zaczepiania bez powodu z waszej strony było już wiele, na tym repo i na repo EI, więc albo się w końcu uspokoicie, albo trzeba będzie was wysłać do psychiatry, żeby wyleczyć wasze trolowanie i hipokryzję, chociaż w sumie niektórzy to powinni się leczyć raczej na nogi bo na głowę już za późno...ewentualnie można na was wezwać hycla

- jesteście doskonałymi przykładami trolli internetowych, a troll to nikt inny jak psychopata o zszarganych nerwach, pełen hipokryzji, który chce innym zszargać nerwy bo mu (trollowi) w życiu się nie ułożyło...

- na koniec diagnozy którą wam postawiłem, ulotka lekarska promująca leki na wasze zszargane nerwy, może w końcu przestaniecie tyle trolować i zaczepiać bez powodu:

![indeks](https://user-images.githubusercontent.com/72980128/99357699-a30dab00-28ac-11eb-97b8-b76967a96a65.jpg) ![bbbbb](https://user-images.githubusercontent.com/72980128/99357781-cafd0e80-28ac-11eb-8500-4ab4de3e4ec6.jpg) ![sssss](https://user-images.githubusercontent.com/72980128/99358330-a7869380-28ad-11eb-998d-5ca43c43a801.jpg)

- na podstawie powyższej diagnozy psychiatryczno-lekarskiej wnoszę o zbanowanie wyżej wymienionych trolli, oni powinni być w szpitalu, a nie na gitubie

**Comment by xxcriticxx** [comment, marked as answer]
@zaba-monika Nikt tobie nie blokowal pisania w dyskusjach. Jak nie działa napisz do @MajkiIT na pewno poprawi.

Z trollowaniem sie nie zgadzam. A za pomoc strasznie dziekuje.

**Reply by zaba-monika** [reply]
- ty nie zablokowałeś wiem, ale hawkeye zablokował, a jeszcze wcześniej ja go pierwsza zablokowałam, ale dlatego bo sobie zasłużył, a on tylko mnie odwetowo zablokował, pomimo, że sam sobie winien bo sam wszczynał zaczepki, no ale skoro chce to niech sobie mnie blokuje, tylko niech wtedy nic nie wypisuje na mnie, bo nie mam jak zripostować na te jego wypociny...
- Z trollowaniem sie nie zgadzasz, ale nie potrafisz uzasadnić dlaczego i podać kontr-argumentu, zaprzeczasz dla samego zaprzeczania...samym tylko pustym gadaniem i "niezgadzaniem się" niczego nie dowiedziesz, i co pewnie  chcesz, żeby ktoś ci na słowo uwierzył? tymczasem dowody na wasze trolowanie są widoczne gołym okiem, wystarczy przeczytać wasze komentarze + mój w tym zgłoszeniu, który opisuje wasze wybryki, wiec możesz się niezgadzać, ale nic ci to nie da, bo i tak udowodniłam, że trolujecie, a ty nie udowodniłeś, że jest inaczej, no ale czego  można się spodziewać po trolach.. jedynie manipulacji, wykręcania się, udawania niewinnych, odwracania kota ogonem i zaprzeczania wszystkiemu za wszelką cenę, zaprzeczaj dalej, że nie trolujecie, możesz nawet zaprzeczać, że 2 + 2 nie równa się 4...tylko kto ci w to uwierzy

**Comment by zaba-monika** [comment]
> https://github.com/MajkiIT/polish-ads-filter/discussions/17449#discussioncomment-135329 :
> xxcriticxx : 21 listopada - Światowy Dzień Życzliwości i Pozdrowień

> hawkeye116477 :
> 
> > Jego idea do dziś jest niezmienna – bądźmy dla siebie życzliwi, rozwiązujmy problemy w sposób pokojowy, a nie siłowy.
>
> Świetna idea, niestety do pewnej osoby obecnej w innym wątku nie przemawia...

> xxcriticxx : good job!

- xxcriticxx jak trolowaliście i zaczepialiście mnie, tak dalej to robicie, pomimo, że pisałem, żebyście się ode mnie odczepili, a ty swojemu koledze przyklaskujesz / przytakujesz, więc nie przychodź tu i nie troluj / nie pisz farmazonów, że "Z trollowaniem sie nie zgadzasz", bo to jest śmieszne, gdy troll troluje, a potem pisze, że nie troluje...

> hawkeye116477 : Świetna idea, niestety do pewnej osoby obecnej w innym wątku nie przemawia...

- znowu trolujesz, czyli ogólnikowe zarzuty bez żadnych konkretów, niby wg. ciebie jestem nieżyczliwa, ale nie wskazałeś o który mój komentarz ci chodzi...więc znowu nie mam do czego się odnieść...sformułuj normalne zarzuty: z linkiem do jakiegoś mojego komentarza, podaj cytat / fragment mojej wypowiedzi / słów, który cię boli i uzasadnij dlaczego cię boli, wtedy będę mogła się jakoś ustosunkować, w tej chwili nie mam co ci odpisać, bo nikt nie wie o co ci chodzi, jakiś bełkot typu "jesteś taka czy sraka" ale nie wiadomo dlaczego...stary weź się ogarnij, ale ty chyba po prostu trolujesz...
- no właśnie, póki co to wy trolujecie mnie, czyli jesteście nieżyczliwi, a nie ja, więc trudno zrozumieć twoje wyssane z palca zarzuty,że to ja jestem nieżyczliwa, wobec kogo niby...świętego mikołaja? No ale to przecież dobrze każdemu znana twoja hipokryzja...

███████████████████████████████████████████████████████████████████████████

- generalnie założyli sobie wątek, w którym jeden mnie troluje, a drugi mu przyklaskuje...nie wiadomo czy się śmiać  czy płakać..
- oni marzą o tym, żeby mnie sprowokować i zdenerwować, żebym zaczęła rzucać wyzwiskami i żeby mieli pretekst, a potem konfidenci szybko zawołają admina z prośbą o bana, ale zaba-monika wie że to prowokacje i nie da się sprowokować, tylko  będzie na spokojnie ripostować z klasą i pokonywać na argumenty i się z nich śmiać
- dodatkowo admin wie, że zaba-monika tu dużo pomaga, więc jest przydatna
